src/main/java/userinterface/InstuctorCreateCourseUserInterface/CreateCourseScreen.java

```diff
@@ -0,0 +1,169 @@
+package userinterface.InstuctorCreateCourseUserInterface;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+
+import javax.swing.JPanel;
+
+import interfaceadapters.CreateCourseInterfaceAdapters.CreateCourseController;
+import interfaceadapters.CreateCourseInterfaceAdapters.CreateCoursePresenter;
+import interfaceadapters.CreateCourseInterfaceAdapters.FailedtoCreate;
+import interfaceadapters.CreateCourseInterfaceAdapters.SuccessCreate;
+import userinterface.GenericProperties;
+
+public class CreateCourseScreen extends JPanel implements ActionListener {
+    private CreateCourseController controller;
+
+    JTextField CourseName = new JTextField(10);
+    JTextField Prerequisite = new JTextField(10);
+    JTextField PrerequisiteGrade = new JTextField(10);
+
+    JTextField Question1 = new JTextField(10);
+
+    JTextField Answer1 = new JTextField(10);
+
+    JTextField Points1 = new JTextField(10);
+
+    JTextField Question2 = new JTextField(10);
+
+    JTextField Answer2 = new JTextField(10);
+
+    JTextField Points2 = new JTextField(10);
+
+    JTextField Question3 = new JTextField(10);
+
+    JTextField Answer3 = new JTextField(10);
+
+    JTextField Points3 = new JTextField(10);
+
+    JButton create = new JButton("Create");
+    JButton cancel = new JButton(("Cancel"));
+    private GenericProperties genericProperties;
+
+    public CreateCourseScreen(GenericProperties genericProperties, CreateCoursePresenter presenter) {
```

Comment:
instead of passing in the presenter to the screen in, initialize it before you retrieve the controller 
something like this CreateCoursePresenter presenter = new CreateCoursePresenter( new DatabaseDriver());
this.controller = presenter.getController(); not a huge change but it makes our main cleaner which follows the feedback we got from pritish